linux/user-and-file-management/file-management/using-find-to-search-by-file-size.md

```diff
@@ -39,7 +39,7 @@ The `find` command is used to search for files on your computer.  It's often use
 This will find all files at least 50MB or larger in the current directory:
```

Comment:
We should also change the `50MB` to say `100MB` in order to match the command bellow (and the rest of the text underneath it).

Reply:
Thanks for spotting that!